src/style.css

```diff
@@ -0,0 +1,20 @@
+
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.h1-heading {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
```

Comment:
`h2` bold by default

Reply:
The "Preamble" and "Article N" must be h2 headings, styled as 15px non-bold.

Reply:
i don't understand the essence of your comment

Reply:
done with h1-heading